rcp/org.testeditor.rcp4.uatests/src/main/java/org/testeditor/fixture/swt/TestEditorFixture.java

```diff
@@ -101,4 +105,12 @@ public String isValidProject(String project) throws CoreException {
 		return Boolean.toString(root.getProject(project).hasNature("org.eclipse.jdt.core.javanature"));
 	}
 
+	@FixtureMethod
+	public String containsWorkspaceFileText(String filePath, String searchText) throws CoreException, IOException {
+		logger.info("Cearching for text {} in {}", searchText, filePath);
+		IWorkspaceRoot root = ResourcesPlugin.getWorkspace().getRoot();
+		return Boolean.toString(Files.lines(new File(root.getLocation().toString() + filePath).toPath())
```

Comment:
That's too much code in one line. Also please reuse existing code, see `org.testeditor.fixture.eclipse.EclipseWorkbenchFixture.getFile(String)`.